src/VirtualClient/VirtualClient.Main/profiles/PERF-MYSQL-OLTP-SYSBENCH.json

```diff
@@ -7,6 +7,7 @@
         "SupportedOperatingSystems": "Debian,Ubuntu"
     },
     "Parameters": {
+        "Benchmark": "OLTP", 
```

Comment:
Remove "Benchmark" parameter from globals. Given the profile specifically (and rightfully) calls out "OLTP-SYSBENCH", do not let the user have an option to override the "Benchmark" on the command line. We do not intend to let the user deviate from the OLTP benchmark when using this profile.